src/main/java/com/jagrosh/jmusicbot/commands/dj/RemoveRangeCmd.java

```diff
@@ -0,0 +1,107 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.audio.QueuedTrack;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.queue.FairQueue;
+
+/**
+ * Command that provides users the ability to remove a range of songs in
+ * the queue.
+ */

```

Comment:
We generally don't use Javadoc comments for commands. In order to stay consistent with this project's code style, remove it.
```suggestion
```